packages/sources/s3reader/src/transport/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { S3Client, GetObjectCommand } from '@aws-sdk/client-s3'
+
+export async function getFileFromS3(
+  s3Client: S3Client,
+  bucket: string,
+  key: string,
+): Promise<string> {
+  const command = new GetObjectCommand({
+    Bucket: bucket,
+    Key: key,
+  })
+
+  const response = await s3Client.send(command)
```

Comment:
I think `s3Client.send` can throw an exception, which we likely want to handle, e.g. for a missing file

Example here: https://github.com/aws/aws-sdk-js-v3/blob/bc2396fd286a5836d6645cf25ba41139da345a52/clients/client-s3/src/waiters/waitForBucketExists.ts#L9-L18

Reply:
Handled in pushed commit